[2025-05-07 12:49:28,705 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read uncommitted';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 12:49:28,707 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 12:49:28,846 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-07 12:49:28,847 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-07 12:55:05,781 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'serializable';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 12:55:05,782 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 12:55:05,869 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-07 12:55:05,870 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-07 13:01:07,192 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'repeatable read';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 13:01:07,193 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 13:01:07,338 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-07 13:01:07,338 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-07 13:07:08,392 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-07 13:07:08,394 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-07 13:07:08,445 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read uncommitted';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 13:07:08,446 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 13:07:29,008 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system reset all;" to update dbms for error: 'NoneType' object has no attribute 'autocommit'
[2025-05-07 13:12:03,648 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read committed';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 13:12:03,650 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 13:17:45,417 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-07 13:17:45,418 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-07 13:17:45,471 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'serializable';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 13:17:45,471 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 13:23:01,273 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-07 13:23:01,274 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-07 13:23:01,326 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'repeatable read';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 13:23:01,327 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 13:23:01,464 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-07 13:23:01,465 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-07 13:23:21,924 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system reset all;" to update dbms for error: 'NoneType' object has no attribute 'autocommit'
[2025-05-07 13:26:50,290 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-07 13:26:50,291 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-07 13:26:50,341 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'repeatable read';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 13:26:50,342 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 13:32:43,191 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read uncommitted';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 13:32:43,193 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 13:33:03,841 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system reset all;" to update dbms for error: 'NoneType' object has no attribute 'autocommit'
[2025-05-07 13:36:54,668 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-07 13:36:54,670 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-07 13:36:54,725 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read committed';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 13:36:54,726 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 13:36:55,120 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-07 13:36:55,121 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-07 13:42:46,889 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read committed';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 13:42:46,890 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 13:48:34,504 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-07 13:48:34,506 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-07 13:48:34,600 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'serializable';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 13:48:34,601 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 13:54:23,181 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-07 13:54:23,183 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-07 13:54:23,273 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'serializable';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 13:54:23,274 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 14:00:22,274 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'serializable';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 14:00:22,276 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 14:06:30,574 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'serializable';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 14:06:30,575 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 14:12:20,403 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'repeatable read';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 14:12:20,404 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 14:18:46,322 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'repeatable read';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 14:18:46,324 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 14:25:09,953 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-07 14:25:09,956 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-07 14:25:10,012 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'serializable';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 14:25:10,012 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 14:31:41,123 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-07 14:31:41,125 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-07 14:31:41,183 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'serializable';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 14:31:41,184 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 14:38:04,926 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-07 14:38:04,927 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-07 14:38:04,984 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'serializable';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 14:38:04,985 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 14:44:48,057 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-07 14:44:48,059 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-07 14:44:48,115 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'repeatable read';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 14:44:48,116 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 14:44:48,264 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-07 14:44:48,265 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-07 14:51:12,387 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read uncommitted';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 14:51:12,388 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 14:57:37,547 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-07 14:57:37,549 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-07 14:57:37,645 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read uncommitted';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 14:57:37,646 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 15:03:30,076 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-07 15:03:30,077 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-07 15:03:30,133 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'serializable';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 15:03:30,134 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 15:03:30,273 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-07 15:03:30,274 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-07 15:09:48,553 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-07 15:09:48,555 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-07 15:09:48,610 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'repeatable read';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 15:09:48,611 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 15:16:13,337 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-07 15:16:13,339 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-07 15:16:13,391 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'serializable';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 15:16:13,391 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 15:16:13,532 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-07 15:16:13,533 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-07 15:22:42,315 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-07 15:22:42,317 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-07 15:22:42,371 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read committed';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 15:22:42,372 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 15:28:55,203 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read committed';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 15:28:55,205 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 15:28:55,409 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-07 15:28:55,409 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-07 15:35:14,601 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read committed';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 15:35:14,603 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 15:41:45,075 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'repeatable read';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 15:41:45,077 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 15:59:29,746 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read committed';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 15:59:29,748 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 16:06:11,182 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-07 16:06:11,184 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-07 16:06:11,234 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read committed';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 16:06:11,235 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 16:06:31,912 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system reset all;" to update dbms for error: 'NoneType' object has no attribute 'autocommit'
[2025-05-07 16:11:43,640 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read committed';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 16:11:43,643 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 16:12:04,623 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system reset all;" to update dbms for error: 'NoneType' object has no attribute 'autocommit'
[2025-05-07 16:17:06,703 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read committed';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 16:17:06,704 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 16:17:27,352 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system reset all;" to update dbms for error: 'NoneType' object has no attribute 'autocommit'
[2025-05-07 16:22:21,951 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-07 16:22:21,953 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-07 16:22:22,009 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read committed';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 16:22:22,010 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 16:22:22,151 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-07 16:22:22,152 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-07 16:22:42,761 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system reset all;" to update dbms for error: 'NoneType' object has no attribute 'autocommit'
[2025-05-07 16:27:36,300 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-07 16:27:36,302 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-07 16:27:36,361 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read committed';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 16:27:36,362 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 16:27:36,512 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-07 16:27:36,513 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-07 16:27:57,020 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system reset all;" to update dbms for error: 'NoneType' object has no attribute 'autocommit'
[2025-05-07 16:32:46,246 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-07 16:32:46,247 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-07 16:32:46,297 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read committed';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 16:32:46,298 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 16:33:07,264 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system reset all;" to update dbms for error: 'NoneType' object has no attribute 'autocommit'
[2025-05-07 16:37:50,430 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-07 16:37:50,432 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-07 16:37:50,490 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read committed';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 16:37:50,491 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 16:38:13,567 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system reset all;" to update dbms for error: 'NoneType' object has no attribute 'autocommit'
[2025-05-07 16:42:29,537 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read committed';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 16:42:29,539 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 16:42:29,856 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-07 16:42:29,857 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-07 16:42:50,354 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system reset all;" to update dbms for error: 'NoneType' object has no attribute 'autocommit'
[2025-05-07 16:47:42,506 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read uncommitted';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 16:47:42,507 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 16:53:52,690 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read committed';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 16:53:52,692 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 16:54:13,787 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system reset all;" to update dbms for error: 'NoneType' object has no attribute 'autocommit'
[2025-05-07 16:58:55,684 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-07 16:58:55,686 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-07 16:58:55,742 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read uncommitted';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 16:58:55,743 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 16:59:16,388 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system reset all;" to update dbms for error: 'NoneType' object has no attribute 'autocommit'
[2025-05-07 17:04:40,563 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read committed';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 17:04:40,564 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 17:04:40,700 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-07 17:04:40,701 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-07 17:10:01,772 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-07 17:10:01,773 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-07 17:10:01,826 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'serializable';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 17:10:01,827 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 17:15:23,309 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-07 17:15:23,311 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-07 17:15:23,473 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'serializable';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 17:15:23,474 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 17:15:23,903 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-07 17:15:23,904 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-07 17:20:51,135 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-07 17:20:51,137 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-07 17:20:51,189 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read committed';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 17:20:51,190 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 17:26:53,453 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read committed';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 17:26:53,455 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 17:26:53,595 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-07 17:26:53,596 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-07 17:33:29,347 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read committed';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 17:33:29,349 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 17:39:56,831 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-07 17:39:56,833 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-07 17:39:56,885 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read committed';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 17:39:56,886 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 17:39:57,026 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-07 17:39:57,027 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-07 17:46:46,960 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-07 17:46:46,962 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-07 17:46:47,014 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'serializable';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 17:46:47,014 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 17:46:47,153 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-07 17:46:47,154 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-07 17:53:23,877 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read committed';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 17:53:23,879 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 17:53:44,512 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system reset all;" to update dbms for error: 'NoneType' object has no attribute 'autocommit'
[2025-05-07 17:58:12,668 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'repeatable read';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 17:58:12,669 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 18:04:13,034 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-07 18:04:13,035 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-07 18:04:13,087 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read uncommitted';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 18:04:13,088 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 18:10:53,267 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read committed';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 18:10:53,269 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 18:17:25,709 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read uncommitted';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 18:17:25,710 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 18:23:43,356 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read committed';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 18:23:43,358 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 18:29:51,855 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-07 18:29:51,857 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-07 18:29:51,912 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read committed';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 18:29:51,913 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 18:29:52,058 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-07 18:29:52,058 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-07 18:36:36,634 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-07 18:36:36,636 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-07 18:36:36,692 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read committed';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 18:36:36,693 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 18:43:38,970 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read uncommitted';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 18:43:38,972 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 18:50:25,282 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read uncommitted';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 18:50:25,284 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 18:57:22,605 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read uncommitted';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 18:57:22,607 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 19:04:01,776 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-07 19:04:01,778 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-07 19:04:01,905 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read committed';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 19:04:01,906 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 19:04:02,327 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-07 19:04:02,328 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-07 19:10:24,269 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-07 19:10:24,271 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-07 19:10:24,328 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read committed';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 19:10:24,329 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 19:16:52,206 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'serializable';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 19:16:52,208 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 19:23:46,002 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-07 19:23:46,003 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-07 19:23:46,056 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read uncommitted';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 19:23:46,057 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 19:30:33,112 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-07 19:30:33,113 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-07 19:30:33,168 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'repeatable read';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 19:30:33,169 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 19:37:06,687 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'serializable';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 19:37:06,689 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 19:43:27,967 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-07 19:43:27,968 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-07 19:43:28,021 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'repeatable read';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 19:43:28,022 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 19:49:43,087 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-07 19:49:43,089 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-07 19:49:43,145 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'serializable';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 19:49:43,145 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 19:55:57,115 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'repeatable read';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 19:55:57,117 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 20:02:34,521 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read committed';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 20:02:34,523 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 20:08:55,021 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read committed';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 20:08:55,023 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 20:08:55,160 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-07 20:08:55,160 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-07 20:15:34,320 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read uncommitted';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 20:15:34,322 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 20:22:13,407 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-07 20:22:13,408 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-07 20:22:13,461 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'repeatable read';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 20:22:13,462 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 20:22:13,603 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-07 20:22:13,604 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-07 20:28:51,572 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read committed';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 20:28:51,574 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 20:28:51,713 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-07 20:28:51,714 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-07 20:35:27,411 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'serializable';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 20:35:27,413 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 20:42:01,747 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'serializable';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 20:42:01,749 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 20:48:31,446 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-07 20:48:31,448 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-07 20:48:31,672 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read uncommitted';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 20:48:31,673 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 20:48:55,778 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system reset all;" to update dbms for error: 'NoneType' object has no attribute 'autocommit'
[2025-05-07 20:53:57,607 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read committed';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 20:53:57,608 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 21:01:02,964 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read committed';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 21:01:02,967 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 21:07:23,206 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read committed';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 21:07:23,207 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 21:14:52,406 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-07 21:14:52,409 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-07 21:14:52,487 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read committed';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 21:14:52,488 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 21:21:55,001 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read committed';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 21:21:55,003 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 21:29:01,570 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-07 21:29:01,572 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-07 21:29:01,630 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read committed';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 21:29:01,630 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 21:29:22,398 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system reset all;" to update dbms for error: 'NoneType' object has no attribute 'autocommit'
[2025-05-07 21:34:27,183 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-07 21:34:27,185 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-07 21:34:27,249 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read committed';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 21:34:27,250 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 21:41:18,534 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read committed';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 21:41:18,535 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 21:41:18,690 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-07 21:41:18,691 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-07 21:41:41,574 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system reset all;" to update dbms for error: 'NoneType' object has no attribute 'autocommit'
[2025-05-07 21:46:38,565 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-07 21:46:38,567 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-07 21:46:38,616 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'serializable';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 21:46:38,617 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 21:53:12,942 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-07 21:53:12,944 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-07 21:53:13,070 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'serializable';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 21:53:13,071 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 21:59:19,235 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-07 21:59:19,237 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-07 21:59:19,288 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'serializable';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 21:59:19,289 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 21:59:19,432 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-07 21:59:19,433 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-07 22:05:52,475 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read uncommitted';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 22:05:52,477 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 22:05:52,622 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-07 22:05:52,623 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-07 22:12:21,897 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read committed';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 22:12:21,898 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 22:19:11,744 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'serializable';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 22:19:11,746 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 22:19:11,891 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-07 22:19:11,892 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-07 22:25:42,881 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-07 22:25:42,882 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-07 22:25:42,931 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read uncommitted';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 22:25:42,933 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 22:25:43,075 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-07 22:25:43,076 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-07 22:32:01,421 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'repeatable read';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 22:32:01,422 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 22:32:01,563 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-07 22:32:01,563 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-07 22:38:40,340 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-07 22:38:40,342 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-07 22:38:40,403 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'serializable';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 22:38:40,404 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 22:45:20,071 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-07 22:45:20,073 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-07 22:45:20,198 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read uncommitted';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 22:45:20,199 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 22:45:20,348 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-07 22:45:20,349 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-07 22:51:46,926 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read uncommitted';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 22:51:46,928 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 22:51:47,082 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-07 22:51:47,083 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-07 22:58:35,874 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-07 22:58:35,876 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-07 22:58:35,935 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read committed';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 22:58:35,936 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 22:58:36,077 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-07 22:58:36,078 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-07 23:05:11,459 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read uncommitted';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 23:05:11,461 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 23:05:11,739 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-07 23:05:11,740 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-07 23:05:32,591 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system reset all;" to update dbms for error: 'NoneType' object has no attribute 'autocommit'
[2025-05-07 23:10:48,261 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read uncommitted';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 23:10:48,263 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 23:10:48,414 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-07 23:10:48,415 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-07 23:17:18,103 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read committed';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 23:17:18,104 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 23:17:18,252 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-07 23:17:18,253 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-07 23:23:57,018 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'repeatable read';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 23:23:57,021 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 23:23:57,163 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-07 23:23:57,164 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-07 23:30:54,287 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read uncommitted';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 23:30:54,288 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 23:31:14,958 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system reset all;" to update dbms for error: 'NoneType' object has no attribute 'autocommit'
[2025-05-07 23:36:31,825 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read uncommitted';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 23:36:31,827 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 23:43:01,240 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read uncommitted';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 23:43:01,242 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 23:43:22,481 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system reset all;" to update dbms for error: 'NoneType' object has no attribute 'autocommit'
[2025-05-07 23:48:25,688 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-07 23:48:25,689 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-07 23:48:25,742 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'serializable';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 23:48:25,743 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 23:48:46,451 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system reset all;" to update dbms for error: 'NoneType' object has no attribute 'autocommit'
[2025-05-07 23:53:38,381 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-07 23:53:38,383 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-07 23:53:38,436 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'repeatable read';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 23:53:38,437 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 23:53:59,164 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system reset all;" to update dbms for error: 'NoneType' object has no attribute 'autocommit'
[2025-05-07 23:58:48,540 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read committed';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-07 23:58:48,542 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-07 23:58:48,696 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-07 23:58:48,697 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
[2025-05-08 00:05:05,576 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set vacuum_cost_limit to '0';" to update dbms for error: 0 is outside the valid range for parameter "vacuum_cost_limit" (1 .. 10000)

[2025-05-08 00:05:05,578 INFO] [postgres.py:set_knob:130] Knob vacuum_cost_limit is set to 200
[2025-05-08 00:05:05,632 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read uncommitted';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-08 00:05:05,633 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-08 00:11:55,463 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set transaction_isolation to 'read uncommitted';" to update dbms for error: parameter "transaction_isolation" cannot be changed

[2025-05-08 00:11:55,465 INFO] [postgres.py:set_knob:130] Knob transaction_isolation is set to read committed
[2025-05-08 00:11:55,641 ERROR] [postgres.py:update_dbms:117] Failed to execute "alter system set autovacuum_freeze_max_age to '0';" to update dbms for error: 0 is outside the valid range for parameter "autovacuum_freeze_max_age" (100000 .. 2000000000)

[2025-05-08 00:11:55,642 INFO] [postgres.py:set_knob:130] Knob autovacuum_freeze_max_age is set to 200000000
